app-admin/tt/tt-0.1.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module
+
+DESCRIPTION="Command-line utility to manage Tarantool applications"
+HOMEPAGE="https://github.com/tarantool/tt"
+SRC_URI="https://github.com/tarantool/tt/releases/download/${PV}/${P}-complete.tar.gz"
+SRC_URI+=" https://github.com/tarantool/tt/releases/download/${PV}/${P}-deps.tar.xz"
+
+LICENSE="BSD-2"
```

Comment:
After our private discussion I thought here a bit more and decided to try to autodeduce licenses. I hit something like https://github.com/mitchellh/golicense/issues/4 with [golicense](https://github.com/mitchellh/golicense), but [lichen](https://github.com/uw-labs/lichen) gives me the following:

```
$ ~/go/bin/lichen tt
github.com/FZambia/tarantool@v0.2.1: BSD-2-Clause (allowed)
github.com/adam-hanna/arrayOperations@v0.2.6: MIT (allowed)
github.com/alecthomas/participle/v2@v2.0.0-alpha4: MIT (allowed)
github.com/apex/log@v1.9.0: MIT (allowed)
github.com/avast/retry-go@v3.0.0+incompatible: MIT (allowed)
github.com/briandowns/spinner@v1.11.1: Apache-2.0 (allowed)
github.com/codeclysm/extract/v3@v3.0.2: MIT (allowed)
github.com/containerd/containerd@v1.5.10: Apache-2.0 (allowed)
github.com/docker/distribution@v2.8.1+incompatible: Apache-2.0 (allowed)
github.com/docker/docker@v20.10.7+incompatible: Apache-2.0 (allowed)
github.com/docker/go-connections@v0.4.0: Apache-2.0 (allowed)
github.com/docker/go-units@v0.4.0: Apache-2.0 (allowed)
github.com/fatih/color@v1.13.0: MIT (allowed)
github.com/fatih/structs@v1.1.0: MIT (allowed)
github.com/gogo/protobuf@v1.3.2: BSD-3-Clause (allowed)
github.com/golang/protobuf@v1.5.2: BSD-3-Clause (allowed)
github.com/h2non/filetype@v1.0.6: MIT (allowed)
github.com/hashicorp/go-version@v1.4.0: MPL-2.0 (allowed)
github.com/hpcloud/tail@v1.0.0: MIT (allowed)
github.com/juju/errors@v0.0.0-20181118221551-089d3ea4e4d5: LGPL-3.0 (allowed)
github.com/mattn/go-colorable@v0.1.12: MIT (allowed)
github.com/mattn/go-isatty@v0.0.14: MIT (allowed)
github.com/mattn/go-pointer@v0.0.1: MIT (allowed)
github.com/mattn/go-runewidth@v0.0.9: MIT (allowed)
github.com/mgutz/ansi@v0.0.0-20200706080929-d51e80ef957d: MIT (allowed)
github.com/mitchellh/mapstructure@v1.4.3: MIT (allowed)
github.com/opencontainers/go-digest@v1.0.0: Apache-2.0, CC-BY-SA-4.0 (allowed)
github.com/opencontainers/image-spec@v1.0.2: Apache-2.0 (allowed)
github.com/otiai10/copy@v1.2.0: MIT (allowed)
github.com/pkg/errors@v0.9.1: BSD-2-Clause (allowed)
github.com/pkg/term@v1.2.0-beta.2: BSD-2-Clause (allowed)
github.com/pmezard/go-difflib@v1.0.0: BSD-3-Clause (allowed)
github.com/shirou/gopsutil@v3.21.2+incompatible: BSD-3-Clause (allowed)
github.com/shurcooL/httpfs@v0.0.0-20190707220628-8d4bc4ba7749: MIT (allowed)
github.com/sirupsen/logrus@v1.8.1: MIT (allowed)
github.com/spacemonkeygo/spacelog@v0.0.0-20180420211403-2296661a0572: Apache-2.0 (allowed)
github.com/spf13/cobra@v1.3.0: Apache-2.0 (allowed)
github.com/spf13/pflag@v1.0.5: BSD-3-Clause (allowed)
github.com/tarantool/go-openssl@v0.0.8-0.20220711094538-d93c1eff4f49: Apache-2.0 (allowed)
github.com/tarantool/go-prompt@v0.2.6-tarantool: MIT (allowed)
github.com/tarantool/go-tarantool@v1.8.1-0.20220921082001-d4905f5de830: BSD-2-Clause (allowed)
github.com/tklauser/go-sysconf@v0.3.4: BSD-3-Clause (allowed)
github.com/tklauser/numcpus@v0.2.1: Apache-2.0 (allowed)
github.com/vmihailenco/msgpack/v5@v5.3.5: BSD-2-Clause (allowed)
github.com/vmihailenco/tagparser/v2@v2.0.0: BSD-2-Clause (allowed)
github.com/yuin/gopher-lua@v0.0.0-20220504180219-658193537a64: MIT (allowed)
golang.org/x/crypto@v0.0.0-20211202192323-5770296d904e: BSD-3-Clause (allowed)
golang.org/x/net@v0.0.0-20211112202133-69e39bad7dc2: BSD-3-Clause (allowed)
golang.org/x/sys@v0.0.0-20210817190340-bfb29a6856f2: BSD-3-Clause (allowed)
golang.org/x/term@v0.0.0-20201126162022-7de9c90e9dd1: BSD-3-Clause (allowed)
google.golang.org/genproto@v0.0.0-20211208223120-3a66f561d7aa: Apache-2.0 (allowed)
google.golang.org/grpc@v1.42.0: Apache-2.0 (allowed)
google.golang.org/protobuf@v1.27.1: BSD-3-Clause (allowed)
gopkg.in/fsnotify.v1@v1.4.7: BSD-3-Clause (allowed)
gopkg.in/natefinch/lumberjack.v2@v2.0.0: MIT (allowed)
gopkg.in/tomb.v1@v1.0.0-20141024135613-dd632973f1e7: BSD-3-Clause (allowed)
gopkg.in/vmihailenco/msgpack.v2@v2.9.2: BSD-3-Clause (allowed)
gopkg.in/yaml.v2@v2.4.0: Apache-2.0, MIT (allowed)
```

```
$ ~/go/bin/lichen --template="{{range .Modules}}{{range .Module.Licenses}}{{.Name | printf \"%s\n\"}}{{end}}{{end}}" tt | sort | uniq -c | sort -nr
     22 MIT
     15 Apache-2.0
     14 BSD-3-Clause
      6 BSD-2-Clause
      1 MPL-2.0
      1 LGPL-3.0
      1 CC-BY-SA-4.0
```

How about replacing this line with the following?

```bash
# The project is licensed under BSD-2, other licenses are from its dependencies.
LICENSE="BSD-2 MIT Apache-2.0 BSD MPL-2.0 LGPL-3"
```

I converted license names to ones used in `/usr/portage/licenses` and excluded `CC-BY-SA-4.0`, because it is a license of README.md and CONTRIBUTING.md files from the [go-digest](https://github.com/opencontainers/go-digest) project (see [here](https://github.com/opencontainers/go-digest#copyright-and-license)) and those files are not part of the installed executable.

I hope that this would be more fair.

Reply:
Everything else seems fine, so I'll just force-push this change and will proceed.